server/src/main/java/com/google/coffeehouse/StorageHandler.java

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Statement;
+
+/**
+* The StorageHandler class holds the query functions to get information from the database.
+*
+* @author  Lina Montes
+* @since   2020-07-06 
+*/
+public class StorageHandler {
+
+  /**
+  * Returns a string containing information about a person queried from the database.
+  * This method formats a string with a user's ID, email, nickname, and pronouns.
+  * If the user has no pronouns (NULL), or the pronouns are empty, then "No pronouns"
+  * is printed out instead.
+  *
+  * @param  dbClient    the database client 
+  * @param  userId      the user ID string used to query and get a person's information
+  * @return personInfo  the formatted string containing the person information
+  */
+  public static String getPersonQuery(DatabaseClient dbClient, String userId) {
+    String personInfo = "";
+    String email = "";
+    String nickname = "";
+    String pronouns = "";
+    Statement statement = 
+        Statement.newBuilder(
+                "SELECT userId, email, nickname, pronouns "
+                  + "FROM Persons "
+                  + "WHERE userId = @userId")
+              .bind("userId")
+              .to(userId)
+              .build();
+    try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+      while (resultSet.next()) {
```

Comment:
Would it be beneficial to check that there's a single result? Currently, it appears that we will return the last result if there are multiple.

Reply:
That's a good point. I changed the while to be an if statement checking to see if the last row in the tables row id is equal to 1, which indicates that there is only 1 entry in the results from the query. There should be no case where there are multiple results because the database only allows for one of each user id, however I will add an else statement to handle when the person does not exist.